pyrsa/util/data_utils.py

```diff
@@ -3,10 +3,8 @@
 """
 Collection of helper methods for data module
 
-+ check_dict_length:   check if each value in dict matches length n
 + extract_dict:        extract key-value pairs with values given indexes.
 + get_unique_unsorted: return a unique unsorted list
-+ check_descriptors_dimension: run check_dict_length and raise an error
 
 @author: baihan
```

Comment:
So you should add yourself to the author list here if we are still doing that. If not let's get rid of them

Reply:
Well, I did not update them anywhere. I would vote for getting rid of them, but didn't do so because I thought some people liked to have them in.